lib/data-processing/storePrAnalysis.ts

```diff
@@ -0,0 +1,26 @@
+import type { AnalyzedPR } from "lib/types"
+import fs from "fs"
+export const storePrAnalysis = (
+  mergedPrsWithAnalysis: AnalyzedPR[],
+  startDate: string,
```

Comment:
thought it was the pr start date

```suggestion
  weekStartDate: string,
```